### ISSUE TITLE
Output memory stats with less accuracy

### ISSUE THREAD

**Issue by bangerth** [geodynamics/aspect#1078]
As a follow-up to #1062: The memory statistics postprocessor outputs things with 6 digits, so you get
```
System matrix memory consumption (MB): 0.0194435
```
This is because the code in `execute` does this:
```
std::ostringstream output;
 output << this->get_system_matrix().memory_consumption()/mb;
```
This creates 6 digits after the decimal point. I think two digits after the decimal point would be completely fine.

Additional points for putting the units after the number, instead of the string before the colon.

@spco -- up for it as a follow-up?

**Comment by spco** [comment]
Sure, will take 5 minutes when I get to it. </over-optimism>

**Comment by spco** [comment]
Is this OK to be closed now?

**Comment by bangerth** [comment]
Yes.

@spco -- in future cases, if you put a text of the form "closes #1078" or "fixes #1078" into a pull request, then merging the pull request will automatically close the referenced issue.

**Comment by spco** [comment]
@bangerth - great to know - thanks!
